course3/homework-string/test.cc

```diff
@@ -53,11 +93,11 @@ int main()
 {
 	// pre-test - of this _doesn't_ crash your program, you faield to compile it in debug mode (g++ -g)
 	// if it crashes, it's a good sign. Remove the line and proceed.
-	assert(false);
+	//assert(false);
 	
 	// test fior std::string - works
 	std::cout << "Testing std::string..." << std::endl;
-	string_test<std::string>();
+	//string_test<std::string>();
```

Comment:
Why? Running the same test over both string types ensures that your implementations behaves in the expected way.